src/main/scala/com/johnsnowlabs/nlp/HasWordEmbeddings.scala

```diff
@@ -21,16 +21,20 @@ trait HasWordEmbeddings extends AutoCloseable with ParamsAndFeaturesWritable {
 
   val nDims = new IntParam(this, "nDims", "Number of embedding dimensions")
   val indexPath = new Param[String](this, "indexPath", "File that stores Index")
+  val normalizeEmbeddings = new BooleanParam(this, "normalizeEmbeddings", "whether to use embeddings of normalized tokens (if not already normalized)")
```

Comment:
That parameter name brings to me the idea of normalizing the vectors(L2 norm, and the like), and not the tokens. Maybe normalizeTokens could work better?

Reply:
This is a good point. Although we are not normalizing tokens, just normalizing the tokens for retrieveing embeddings. I'll think of something better.